Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -348,13 +358,29 @@ if (rc == 0) {
 		println("** There are no build outputs found in all provided build reports. Exiting.")
 		rc = 0
 	} else {
-	
+		// generate SBOM only if build outputs exist
+		if (props.generateSBOM && props.generateSBOM.toBoolean()) {
+			sbomUtilities = loadScript(new File("${scriptDir}/utilities/sbomGenerator.groovy"))
+			sbomSerialNumber = "url:uuid:" + UUID.randomUUID().toString()
+			sbomFileName = "${buildNumber}_sbom.json" 
```

Comment:
Is the file name format rather something that we should declare at the beginning of the script?

Reply:
It looks like the buildNumber is retrieved from the build report. So, it might make sense to retain this where it is.